frontend/src/ui/strings.json

```diff
@@ -107,6 +110,9 @@
     "becomeMemberTitle": "Pourquoi devenir un supporter?",
     "becomeMemberText": "Avec votre contribution, vous nous aidez à couvrir nos dépenses. Tous les fonds sont gérés de manière responsable et en toute transparence. Tout le monde peut voir comment on dépense l'argent récolté.",
     "howWeSpend": "Voir comment on utilise l'argent",
-    "donateWithPayPal": "Payer avec PayPal"
+    "donateWithPayPal": "Payer avec PayPal",
+    "and": "et",
+    "weHave": "Nous avons",
+    "thatProvideYearlyBudget": "qui nous fournissent avec un budget annuel de"
```

Comment:
ah ah nice try :-D

(almost!)
Should be "qui nous fournissent un budget annuel de"